docs/source/samples/14_2_mono_camera_control.rst

```diff
@@ -0,0 +1,37 @@
+14.2 - Mono Camera Control
+=========================
```

Comment:
for some reason (there's a warning when building docs) length of === has to be the same as the title, so add another "="

Reply:
fixed it in every doc